### DISCUSSION TITLE
Version@3.4.8

### DISCUSSION THREAD

**Discussion by khengyun** [khengyun/autonext#24] [Announcements]
Download For Window: [click here](https://github.com/khengyun/autonext/archive/refs/tags/v3.4.8.zip)
Download For Linux: [click here](https://github.com/khengyun/autonext/archive/refs/tags/v3.4.8.tar.gz)

Feature:

Presentation Grading
Group Members Grading

Update Check New Version

[NEW]: 
Support Chromium and Microsoft Edge

v3.4.8

<hr /><em>This discussion was created from the release <a href='https://github.com/khengyun/autonext/releases/tag/v3.4.8'>Version@3.4.8</a>.</em>

**Comment by mmotkim** [comment]
Link download for windows trỏ đến file v3.4.6 r dev ơi

**Reply by khengyun** [reply]
đã sửa, cảm ơn bạn nhé
